### PR TITLE
Remove unperformant dashboard show more stats

### PR DESCRIPTION
**Pull request by unho**
This is rarely used, but when used it has a big impact on server
performance.

References #5016.

### CODE REVIEW

File: pootle/apps/pootle_app/views/admin/urls.py [hunk 1]
```diff
@@ -50,9 +50,9 @@
         name='pootle-admin-permissions'),
 
     # XHR
-    url(r'^more-stats/?$',
-        dashboard.server_stats_more,
-        name='pootle-admin-more-stats'),
+#    url(r'^more-stats/?$',
+#        dashboard.server_stats_more,
+#        name='pootle-admin-more-stats'),
```
Comment (phlax) [inline pootle/apps/pootle_app/views/admin/urls.py]: dont comment out - we can add it back if we need it - but no point distributing commented code

**Comment by phlax** [comment]
@unho would this stop the link from showing?

**Comment by unho** [comment]
Removed the code instead of commenting it, and also removed the link.

**Comment by julen** [comment]
If disabled, I'd be happy to see the code go away—any proper implementation will very likely be different anyway.

Nit: this is not a speedup on its own, but prevents an accidental resource hog.

**Comment by unho** [comment]
@julen Updated to comply with your requests.

**Comment by phlax** [comment]
lgtm, nice one @unho

**Comment by julen** [comment]
You can also remove the [`a.slide` style declaration](https://github.com/translate/pootle/blob/f7c601d96271e82ef2182e2149945ea8466cae12/pootle/static/css/style.css#L2054). GTM.

**Pull request merged**